src/scss/packages/image-example/src/_styles.scss

```diff
@@ -0,0 +1,26 @@
+@use 'uswds-core' as *;
+
+.image-example .bottom-border-green {
+  border-bottom: 0.5rem solid $theme-color-success;
+}
+
+.image-example .bottom-border-red {
+  border-bottom: 0.5rem solid $theme-color-error;
+}
+
+.image-example .text-green {
+  color: $theme-color-success;
+}
+
+.image-example .text-red {
+  color: $theme-color-error;
+}
+
+.image-example figure {
+  display: table;
+}
+
+.image-example figcaption {
+  display: table-caption;
+  caption-side: bottom;
+}
```

Comment:
This is the cleanest way I found to make the text wrap under the image correctly. To see what I mean, try wrapping this whole component in divs of various widths:
```html
<div style="width:200px" >
  <div class="image-example">
    ...
  </div>
</div>
```
And
```html
<div style="width:1200px" >
  <div class="image-example">
    ...
  </div>
</div>
```

Without these rules, the caption will overflow the image's width at the larger sizes.